llama_toolchain/cli/stack/configure.py

```diff
@@ -40,40 +41,144 @@ def _add_arguments(self):
         self.parser.add_argument(
             "config",
             type=str,
-            help="Path to the package config file (e.g. ~/.llama/builds/<distribution>/<image_type>/<name>.yaml)",
+            help="Path to the build config file (e.g. ~/.llama/builds/<distribution>/<image_type>/<name>-build.yaml)",
         )
 
     def _run_stack_configure_cmd(self, args: argparse.Namespace) -> None:
         from llama_toolchain.core.package import ImageType
 
-        config_file = Path(args.config)
-        if not config_file.exists():
-            self.parser.error(
-                f"Could not find {config_file}. Please run `llama stack build` first"
+        with open(args.config, "r") as f:
+            try:
+                build_config = BuildConfig(**yaml.safe_load(f))
+            except Exception as e:
+                self.parser.error(
```

Comment:
This message is not correct. Because the exception can happen due to many other reasons than just not being able to find the file. Please revert to the previous behavior where we explicitly test for file existence.